### ISSUE TITLE
DeDRM from new Barnes and Noble books not working

### ISSUE THREAD

**Issue by timlt** [apprenticeharper/DeDRM_tools#794]

**Comment by timlt** [comment]
Hi, I just opened a BN account, downloaded latest Nook app from Windows store, and bought a few books.  I have Calibre with latest DeDRM plugins.  I could DeDRM two of the olders books and and a free one, but two did not work.  These were newer titles by Roger Scruton.  I tried both methods of getting a key:  I put in username and password in the plugin, fails.  I tried searching for *.b64 key files on my PC, nothing.  Even searched around in some of the BN config files for possible keys, couldn't find anything.  Read FAQ at Alf's blog, couldn't find a workaround.

I have returned those books so I don't think it's possible now to debug my instance of this issue.  But before someone closes it, can you suggest what MIGHT be the issue or workaround here?  If BN has changed the encryption scheme again, and broken DeDRM on new books in the current Nook app for Win10, I am just going to buy books elsewhere until the DeDRM works again.  Thanks.

**Comment by ElleKayEm** [comment]
In the past, I found that some books downloaded with the Windows app (comics, magazines) would have part of their DRM stored outside the epub itself in a separate database.  Possibly they've started doing this with more or all of their books.  As you said, we can't really troubleshoot with books you've already returned.  But you can take a look at another issue where I explain how I got my Nook comics DeDRMed.  As you'll see, it didn't work for some others.

https://github.com/apprenticeharper/DeDRM_tools/issues/693#issuecomment-442184522

**Comment by grillod** [comment]
I don't know how you got it to work at all in light of this issue https://github.com/apprenticeharper/DeDRM_tools/issues/754 which is the one I'm still having problems with.

**Comment by ElleKayEm** [comment]
I've seen a couple people report they didn't need to customize the plugin with their B&N info when using the Windows 8/10 Nook app.  So apparently DeDRM can retrieve the key from it, just as it did with Nookstudy...or at least it could.

**Comment by grillod** [comment]
Yes youre right. That seems to be the case. I gave up after customization failed. But importing does work on some of my mew books bit not others. Thanks!

**Comment by timlt** [comment]
@grillod yep that's my experience.  But I don't want a hit-and-miss solution, and don't have time to mess with it.  So for now, it's use other stores.
